src/components/UiSelect/UiSelectOptions.vue

```diff
@@ -60,15 +60,15 @@ export default {
   computed: {
     filteredOptions () {
       return this.items.filter(option => option.show && !option.hidden)
-    }
+    },
```

Comment:
remove extra commas everywhere you added

Reply:
fixed